optimum/habana/peft/layer.py

```diff
@@ -29,3 +31,41 @@ def GaudiLoraLayerLinearForward(self, x: torch.Tensor, *args: Any, **kwargs: Any
 
     result = result.to(previous_dtype)
     return result
+
+class LoRALinear:
+    def __init__(self, module):
+        has_bias = module.bias is not None
+        self.module = module
+        self.module.te_linear =  te.Linear(module.in_features, module.out_features, bias=has_bias, params_dtype=module.weight.dtype, skip_weight_param_allocation=True)
```

Comment:
This got created without parameters, where do actual parameters get hooked to it?

Reply:
Done in _linear function (Line 42).